[2025-10-06T21:32:06.711+0000] {processor.py:161} INFO - Started process (PID=41) to work on /opt/airflow/dags/main.py
[2025-10-06T21:32:06.718+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:32:06.726+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:06.725+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:32:06.753+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:06.750+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:32:06.754+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:32:06.785+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.082 seconds
[2025-10-06T21:32:37.185+0000] {processor.py:161} INFO - Started process (PID=51) to work on /opt/airflow/dags/main.py
[2025-10-06T21:32:37.185+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:32:37.189+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:37.189+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:32:37.201+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:32:37.199+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:32:37.201+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:32:37.223+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.068 seconds
[2025-10-06T21:33:07.622+0000] {processor.py:161} INFO - Started process (PID=61) to work on /opt/airflow/dags/main.py
[2025-10-06T21:33:07.623+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:33:07.626+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:07.626+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:33:07.632+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:07.632+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:33:07.633+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:33:07.661+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.057 seconds
[2025-10-06T21:33:38.001+0000] {processor.py:161} INFO - Started process (PID=71) to work on /opt/airflow/dags/main.py
[2025-10-06T21:33:38.002+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:33:38.004+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:38.004+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:33:38.012+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:33:38.010+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:33:38.013+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:33:38.052+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.064 seconds
[2025-10-06T21:34:08.409+0000] {processor.py:161} INFO - Started process (PID=81) to work on /opt/airflow/dags/main.py
[2025-10-06T21:34:08.411+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:34:08.414+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:08.414+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:34:08.419+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:08.418+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:34:08.420+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:34:08.444+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.049 seconds
[2025-10-06T21:34:38.792+0000] {processor.py:161} INFO - Started process (PID=91) to work on /opt/airflow/dags/main.py
[2025-10-06T21:34:38.794+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:34:38.806+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:38.803+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:34:38.826+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:34:38.825+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:34:38.826+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:34:38.862+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.093 seconds
[2025-10-06T21:35:09.257+0000] {processor.py:161} INFO - Started process (PID=101) to work on /opt/airflow/dags/main.py
[2025-10-06T21:35:09.259+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:35:09.263+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:09.263+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:35:09.268+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:09.267+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:35:09.268+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:35:09.292+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.074 seconds
[2025-10-06T21:35:39.657+0000] {processor.py:161} INFO - Started process (PID=111) to work on /opt/airflow/dags/main.py
[2025-10-06T21:35:39.660+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:35:39.664+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:39.663+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:35:39.671+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:35:39.670+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:35:39.672+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:35:39.696+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.050 seconds
[2025-10-06T21:36:10.263+0000] {processor.py:161} INFO - Started process (PID=121) to work on /opt/airflow/dags/main.py
[2025-10-06T21:36:10.265+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:36:10.267+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:10.267+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:36:10.285+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:10.284+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:36:10.287+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:36:10.311+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.089 seconds
[2025-10-06T21:36:40.800+0000] {processor.py:161} INFO - Started process (PID=131) to work on /opt/airflow/dags/main.py
[2025-10-06T21:36:40.805+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:36:40.810+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:40.809+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:36:40.822+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:36:40.820+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:36:40.823+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:36:40.849+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.095 seconds
[2025-10-06T21:37:11.289+0000] {processor.py:161} INFO - Started process (PID=141) to work on /opt/airflow/dags/main.py
[2025-10-06T21:37:11.294+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main.py for tasks to queue
[2025-10-06T21:37:11.300+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:37:11.299+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main.py
[2025-10-06T21:37:11.311+0000] {logging_mixin.py:188} INFO - [2025-10-06T21:37:11.309+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/main.py", line 6, in <module>
    from airflow.providers.standard.operators.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T21:37:11.312+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main.py
[2025-10-06T21:37:11.342+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main.py took 0.071 seconds
